src/main/java/tutorease/address/model/LessonSchedule.java

```diff
@@ -47,21 +47,39 @@ public boolean hasLesson(Lesson lesson) {
     }
 
     /**
-     * Adds a lesson to the lesson schedule.
-     * The lesson must not already exist in the lesson schedule.
+     * Adds the specified lesson to the lesson list.
+     *
+     * @param lesson The lesson to be added. Must not be null.
+     * @throws NullPointerException if the specified lesson is null.
```

Comment:
not sure if needed but maybe can caps the if so that it is If

Reply:
yea i thought so too but our textbook suggested that
![image](https://github.com/user-attachments/assets/3375bec3-b199-42d3-816e-4e9326e233e0)

Reply:
fixed